azurerm/resource_arm_virtual_machine_scale_set.go

```diff
@@ -396,7 +396,7 @@ func resourceArmVirtualMachineScaleSet() *schema.Resource {
 
 									"primary": {
 										Type:     schema.TypeBool,
-										Optional: true,
+										Required: true,
```

Comment:
as this is now required - can we update this in [the documentation](https://github.com/terraform-providers/terraform-provider-azurerm/blame/master/website/docs/r/virtual_machine_scale_set.html.markdown#L377) too?

Reply:
Done!